src/System.Management.Automation/engine/hostifaces/PSTask.cs

```diff
@@ -456,7 +456,7 @@ public void Start()
 
             // Create the PowerShell command pipeline for the provided script block
             // The script will run on the provided Runspace in a new thread by default
-            _powershell = PowerShell.Create(_runspace);
+            _powershell = PowerShell.Create(runspace);
```

Comment:
There is no difference in using the parameter argument or the class field, since they are the same.  The field is used to pass the task runspace back to the pool later.